sd.go

```diff
@@ -73,6 +78,38 @@ func LookupSidByName(name string) (sid string, err error) {
 	return sid, nil
 }
 
+// LookupNameBySid looks up the name of an account by SID
+func LookupNameBySid(sid string) (name string, err error) {
+	if sid == "" {
+		return "", &AccountLookupError{sid, cERROR_NONE_MAPPED}
+	}
+
+	sidBuffer, err := syscall.UTF16FromString(sid)
```

Comment:
I'd prefer the x/sys/windows variants for any syscall usage in this function. I know we're not great about that in this project at the moment but we should switch over soon

Reply:
Fixed.